am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetRolePermissionsIntegrationTest.java

```diff
@@ -42,13 +46,23 @@ void setUp() {
 
     @Test
     void whenGettingRolePermissionsShouldReturnPermissionsAndSecurityClassificationsForSpecifiedRole() {
+
+        ImmutableSet multiPermission = ImmutableSet.builder().add(READ).add(CREATE).add(UPDATE).add(DELETE).build();
```

Comment:
Change immutable set to interface to stick to the rule advising coding against interfaces.

Reply:
Why is that extracted to variable if other sets are not? Be consistent, I do not think you need that variable really.

Reply:
done